server/service/src/sync/sync_user.rs

```diff
@@ -0,0 +1,85 @@
+use chrono::NaiveDateTime;
+use log::info;
+use repository::{RepositoryError, UserAccountRow, UserAccountRowRepository};
+
+use crate::{
+    auth_data::AuthData,
+    login::{FetchUserError, LoginError, LoginInput, LoginService},
+    service_provider::ServiceProvider,
+};
+
+pub struct SyncUser {}
+
+impl SyncUser {
+    pub async fn update_user(
```

Comment:
I think a small comment would be good here:
// Re-login to central server with user credentials to update latest user info (mainly user permissions)